torchvision/transforms/functional.py

```diff
@@ -471,6 +485,8 @@ def resize(
     if (image_height, image_width) == output_size:
         return img
 
+    antialias = _check_antialias(img, antialias, interpolation)
+
     if not isinstance(img, torch.Tensor):
         if antialias is not None and not antialias:
             warnings.warn("Anti-alias option is always applied for PIL Image input. Argument antialias is ignored.")
```

Comment:
Yes, `is False` is clearer, althought I avoided doing this in this PR because it's not strictly needed for the original purpose, and the PR is already pretty long. I don't mind doing if you think it's less confusing

Reply:
I don't care if we do it here or in a follow-up, but we should do it. I had to build the table above to see if the new case of `"warn"` does something unexpected and that is not a good sign for an otherwise simple check.

Reply:
@pmeier `antialias="warn"` should behave as `antialias=None`, so, I would say there is no 4th line in the table (also values of the 4th line seem to be incorrect).

Reply:
Well, the result is the same, but the way it is achieved is different. What do you think is wrong?

```pycon
>>> antialias = "warn"
>>> antialias is not None
True
>>> not antialias
False
>>> antialias is not None and not antialias
False
```